### ISSUE TITLE
Documentation for CRON job

### ISSUE THREAD

**Issue by J-R-Mann** [sdanielzafar/NAVtrackr#6]
I would like to see some documentation/functionality to have this as a scheduled CRON job. Or maybe that is just something totally different and not part of NAV trackr. Either way, I would prefer that the code just runs on Sunday evening.

**Comment by sdanielzafar** [comment]
agreed, that would be nice. I think that this would need to come out at the same time as `report_delete()` because without a doubt there will be Sundays when your timesheet is submitted but the toggl accuracy was trash so users would need an easy way to delete the previous timesheet and resubmit.
